Dockerfile-datamgr

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:focal
+FROM gcr.io/distroless/base:latest
 RUN apt-get update && \
```

Comment:
I believe gcr.io/distroless/base-debian10 includes the certificates packages, so you could remove the apt-get

Reply:
Removed apt-get part.